ddtrace/bootstrap/sitecustomize.py

```diff
@@ -81,6 +81,14 @@ def update_patched_modules():
     if profiling:
         import ddtrace.profiling.auto  # noqa: F401
 
+    if asbool(get_env("appsec", "enabled", default=False)):
+        # Keep the import local until the module is enabled by default
+        import ddtrace.appsec
+
+        config.http.hooks.register("request", ddtrace.appsec.process_request)
```

Comment:
Hmm not 100% about whether we want to reuse the config api for this. It does totally make sense right now but just uncertain if we want to keep adding stuff to this api.

Also given how light a change this is I'm not too worried either.

Reply:
Agreed! I removed the HTTP hooks on the config interface and added a static call to the AppSec's `process_request` method instead. Only the Flask instrumentation is modified for now until we figure out a good enough event based solution.